tools/haskell-translator/lhs_pars.py

```diff
@@ -980,7 +980,7 @@ def type_wrapper_type(header, cons, rhs, d, decons=None):
             % (decons, header, header, header, header),
             'where',
             '  %s_update_def[simp]:' % decons,
-            ' "%s_update f y \<equiv> f y"' % decons,
+            ' "%s_update f y \\<equiv> f y"' % decons,
```

Comment:
any reason behind this choice? testing on python3:
```
>>> r'test %s' % 3
'test 3'
```

seems to work. I don't really care that much, but I'm curious.

Reply:
I find it clearer to separate these -- the point of raw string literals is that there are no special characters. I guess the `%s` bit works because it's an implicit call to `format` on a string that is by then arbitrary (doesn't matter any more what kind of literal it was), but that's a bunch of speculation on my part. This way it's definitely clear what's happening, with `r` I would eternally be unsure when reading it.